config.py

```diff
@@ -0,0 +1,74 @@
+import configparser
+from os import path
+
+conf_file = path.join(path.dirname(__file__), 'config.ini')
+
+
+class Config:
+    def __init__(self, section):
+        self._parser = configparser.RawConfigParser()
+        self._config_file = conf_file
+        self._section = ''
+        try:
+            self._parser.read(self._config_file)
+        except configparser.Error, exc:
+            raise configparser.Error("Error reading config file %r: %s" %
+                                     (self._config_file, str(exc)))
+
+        if not self._parser.has_section(section):
+            print 'Available sections:'

```

Comment:
Please, don't use prent, this cause for logger =)

Reply:
Re-write this sentence correctly, not in Indian style, please =)

Reply:
Please, don't use _print_
Using of logger is better here
:-P